keyboards/ergodox/infinity/drivers/gdisp/IS31FL3731C/gdisp_IS31FL3731C.c

```diff
@@ -204,7 +204,8 @@ LLDSPEC bool_t gdisp_lld_init(GDisplay *g) {
 		uint8_t* src = PRIV(g)->frame_buffer;
 		for (int y=0;y<GDISP_SCREEN_HEIGHT;y++) {
 		    for (int x=0;x<GDISP_SCREEN_WIDTH;x++) {
-		        PRIV(g)->write_buffer[get_led_address(g, x, y)]=CIE1931_CURVE[*src];
+		        uint8_t val = (uint16_t)*src * g->g.Backlight / 100;
```

Comment:
Thanks for testing, this gives some hints where to look further. The `src` should have the right range already (0-255), actually the uGFX `GDISP_PIXELFORMAT_GRAY256` format. 

The scaling by 100, is because the backlight level is represented as percentage.

I think the problem might be that `gdispColor2Native` does not do the correct thing in the `gdisp_lld_draw_pixel`  function. Or perhaps that there is some other bug and draw pixel is never called.